src/Chapter1.hs

```diff
@@ -653,8 +665,10 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
-
+firstDigit :: Int -> Int
+firstDigit x
+  | x `div` 10 == 0 = x
+  | otherwise       = firstDigit (x `div` 10)
```

Comment:
I see, that for a negative numbers it will be not return the correct answer. Sometimes it is tricky to keep in mind all the edge cases, so no worries, but the idea and recursive solution is awesome 👏🏼